linkme-chatapp/devServer.js

```diff
@@ -126,6 +137,15 @@ function createApp() {
 
     return false;
   }
+  
+  function validateFacebookUserToken(userId, userToken, onResponse) {
```

Comment:
Right now I think you're only calling the onResponse callback if you succeded. Might want to call regardless 😄